### PR TITLE
similar -> similarly

### PR DESCRIPTION
**Pull request by AlvaroBurnett**

### CODE REVIEW

**Review by ISSOtm — CHANGES_REQUESTED** [review]
s/opcodes/instructions/gi

File: content/Memory_Map.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The following addresses are supposed to be used as jump vectors:
 
 However, the memory may be used for any other purpose in case that your
 program doesn't use any (or only some) RST commands or interrupts. RST
-commands are 1-byte opcodes that work similar to CALL opcodes, except
+commands are 1-byte instructions that work similarly to CALL instructions, except
```
Comment (ISSOtm) [inline content/Memory_Map.md:30]: I'd rather say that "`rst` is a 1-byte instruction that works similarly to the 3-byte `call` instruction, except that the destination address is restricted.". What do you think?

**Review by ISSOtm — CHANGES_REQUESTED** [review]

File: content/Memory_Map.md (hunk 1 above)
Comment (AlvaroBurnett) [reply content/Memory_Map.md]: I agree. Fixed.

**Review by ISSOtm — APPROVED** [review]
Will wait on others' feedback to be sure about readability.

**Review by avivace — APPROVED** [review]

**Pull request merged**